dev/index.html

```diff
@@ -185,6 +190,7 @@ <h1>Manager</h1>
           height="500"
           display-start="5"
           display-end="25"
+          hm-highlight-width="0"
```

Comment:
If I understand correctly, the only difference in not adding this attribute is the border of the highlighted area defaults to 1, isn't it?

Reply:
Yes. This was added so the Nightingale Sequence Heatmap matches the highlighted style of Nightingale better after the PDBe heatmap-component fixes